### PR TITLE
UI adaptable to desktop version

### PR DESCRIPTION
**Pull request by emmanuelcortes2**
### Desktop version & responsiveness

- Add media queries to style.css so it displays correctly to different screen sizes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,119 +53,131 @@ <h4>LET'S CONNECT</h4>
...
+                        <li class="element2">2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.</p>
+                    <ul class="bottom-list">
+                        <li><button class="language-buttons" type="button">html</button></li>
+                        <li><button class="language-buttons" type="button">css</button></li>
+                        <li><button class="language-buttons" type="button">javaScript</button></li>
+                    </ul>
+                    <button class="p-button" type="button">See project</button>
```
Comment (awais-amjed) [inline index.html:141]: Kindly update the button hover to show the text in white as shown in Figma design 😃

Your design:

image

Figma design:

image

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @emmanuelcortes2 , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] No Linter issues ✔️ 
- [x] Nice and Responsive Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -181,7 +200,7 @@ <h4>LET'S CONNECT</h4>
                 </ul>
                 </nav>
         </section>
-        <footer class="contact">
+        <footer class="contact" id="contact">
```
Comment (vickymarz) [inline index.html:203]: - [ ] I think you kind of introduced an overflow in your contact section. Please adjust it to match the Figma design template chosen 👍🏽 

Your Design 👇🏽 

image

Figma Design 👇🏽 

image

File: index.html [hunk 3]
```diff
@@ -47,119 +54,131 @@ <h4>LET'S CONNECT</h4>
...
-                <p>A daily selection of privately personalized reads; 
-                    no accounts or sign-ups required.</p>
-                <ul class="bottom-list">
-                    <li><button class="language-buttons" type="button">html</button></li>
-                    <li><button class="language-buttons" type="button">css</button></li>
-                    <li><button class="language-buttons" type="button">javaScript</button></li>
-                </ul>
-                <button class="p-button" type="button">See project</button>
+                <div class="column">
+                    <h1><strong>Tonic</strong></h1>
```
Comment (vickymarz) [inline index.html:64]: - [ ] As the previous reviewer suggested, kindly increase the font weight of all your project card heading to match the Figma design 🤗.

Your design 👇🏽 

image

Figma design 👇🏽 

image

File: index.html [hunk 4]
```diff
@@ -47,119 +54,131 @@ <h4>LET'S CONNECT</h4>
...
-                    <li>
-                        <a href="#"><img src="peaceandlove.svg" alt="peace"></a>
-                    </li>
-                </ul>
-            </div> 
-            <div>
-                <button class="p-button" type="button">Get my resume</button>
+        <section class="about-me" id="about-me">
+            <div class="column1">
+                <h1><strong>About<br>Myself</strong></h1>
```
Comment (vickymarz) [inline index.html:148]: - [ ] As the previous reviewer suggested, kindly increase the font weight of your **_` About Myself`_** heading to match the Figma design template you chose🤗 and also use the right color specified in the Figma design as your color is currently black.

Your design 👇🏽 

image

Figma design 👇🏽 

image

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @emmanuelcortes2 , 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @emmanuelcortes2, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations


## Good Highlights ✅

- Implemented changes recommended
- Adherence to GitHub flow
- Descriptive PR
- Good commit messages

## Optional suggestions

- I recommend that you organize your images into a folder in order to reduce the cluttering of your project's home directory.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**